Sources/Commands/SwiftTool.swift

```diff
@@ -530,7 +530,8 @@ public class SwiftTool {
             isResolverPrefetchingEnabled: options.shouldEnableResolverPrefetching,
             skipUpdate: options.skipDependencyUpdate,
             enableResolverTrace: options.enableResolverTrace,
-            cachePath: cachePath
+            cachePath: cachePath,
+            verbosity: isVerbose ? .verbose : .concise
```

Comment:
should this use options.verbosity?

Reply:
`ToolWorkspaceDelegate` got the `isVerbose` as an initializer parameter so I guess this is more important than options. In general, I agree that this should be in options however, that's not where the SwiftTool is yet, and I'd prefer to split that in another PR. As of now, the global variable is still in use by the CLI code.